docs/public/specifications/tests/Connectivity-TP/connectivity04.md

```diff
@@ -50,12 +48,14 @@ giving a correct DNS response for an authoritative name server.
 
 Message Tag                 | Level   | Arguments                   | Message ID for message tag
 :---------------------------|:--------|:----------------------------|:------------------------------------------------------------------------------------------------
-CN04_EMPTY_PREFIX_SET       | ERROR   | ns_ip                       | Prefix database returned no information for IP address {ns_ip}.
-CN04_ERROR_PREFIX_DATABASE  | ERROR   | ns_ip                       | Prefix database error. No data to analyze for IP address {ns_ip}.
-CN04_IPV4_SAME_PREFIX       | WARNING | ns_list, ip_prefix          | The following name server(s) are announced in the same IPv4 prefix ({ip_prefix}): "{ns_list}"
+CN04_EMPTY_PREFIX_SET       | NOTICE  | ns_ip                       | Prefix database returned no information for IP address {ns_ip}.
+CN04_ERROR_PREFIX_DATABASE  | NOTICE  | ns_ip                       | Prefix database error for IP address {ns_ip}.
 CN04_IPV4_DIFFERENT_PREFIX  | INFO    | ns_list                     | The following name server(s) are announced in unique IPv4 prefix(es): "{ns_list}"
-CN04_IPV6_SAME_PREFIX       | WARNING | ns_list, ip_prefix          | The following name server(s) are announced in the same IPv6 prefix ({ip_prefix}): "{ns_list}"
+CN04_IPV4_SAME_PREFIX       | NOTICE  | ns_list, ip_prefix          | The following name server(s) are announced in the same IPv4 prefix ({ip_prefix}): "{ns_list}"
+CN04_IPV4_SINGLE_PREFIX     | WARNING |                             | All name server(s) (IPv4) are announced in the same IPv4 prefix.
 CN04_IPV6_DIFFERENT_PREFIX  | INFO    | ns_list                     | The following name server(s) are announced in unique IPv6 prefix(es): "{ns_list}"
```

Comment:
The change of logic was not intentionally. I rather revert so that one such message is outputted per test. It does not add any value to output the message for each name server, especially not when the message is on INFO level.